rust-install/src/mock/mod.rs

```diff
@@ -1,3 +1,10 @@
+//! Mocks for testing
+//!
+//! FIXM: This belongs in some other crate
+
```

Comment:
Doesn't seem like it can. The tests don't resolve if I put `#[cfg(test)]` on it.